discovery-provider/src/tasks/index.py

```diff
@@ -457,7 +457,7 @@ def refresh_peer_connections(task_context):
             stored_in_redis = redis.get(cnode_url)
             if stored_in_redis is not None:
                 continue
-            if cnode_url == '':
+            if cnode_url == "''":
```

Comment:
should this also get fixed on line 443?

Reply:
this is actually coming from https://github.com/AudiusProject/audius-protocol/pull/735/files#diff-dfa98e3ab36bd75592fbfb94f694400cR448-R449 here are the logs associated with it
```
[2020-08-06 15:45:59,161] {src.tasks.index:453} (WARNING) - the user_node_url
[2020-08-06 15:45:59,162] {src.tasks.index:454} (WARNING) - ''
```

I think it'd be good to consolidate the logic in one area here. maybe actually removing line 443 would accomplish the consolidating

Reply:
ah i see this is the user_node_url, that makes sense! isn't it still _possible_ to get a `''` string literal in the cnode endpoint set? if a user has `cnode1,,cnode2,cnode3` for whatever reason won't we get `''`. so i actually think 443 is still needed and still needs this check

Reply:
also, what happens if we actually do get a None type object as the value for user_metadata?

Reply:
just checked, and the `None` type gets converted into the literal single quotes like this
```
{'': True}
```

in this case, this PR will be able to handle this 😃

Reply:
does this address the lines on 443? seems like we can still have the same issue there

Reply:
just added it!